src/main/java/com/faforever/client/preferences/ChatPrefs.java

```diff
@@ -2,24 +2,42 @@
 
 import com.faforever.client.chat.ChatColorMode;
 import com.faforever.client.chat.ChatFormat;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+
 import javafx.beans.property.BooleanProperty;
 import javafx.beans.property.DoubleProperty;
 import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.ListProperty;
 import javafx.beans.property.MapProperty;
 import javafx.beans.property.ObjectProperty;
 import javafx.beans.property.SimpleBooleanProperty;
 import javafx.beans.property.SimpleDoubleProperty;
 import javafx.beans.property.SimpleIntegerProperty;
+import javafx.beans.property.SimpleListProperty;
 import javafx.beans.property.SimpleMapProperty;
 import javafx.beans.property.SimpleObjectProperty;
 import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
 import javafx.collections.ObservableMap;
 import javafx.scene.paint.Color;
 
+import java.util.Locale;
+import java.util.Optional;
+
 import static com.faforever.client.chat.ChatColorMode.CUSTOM;
+import static com.faforever.client.preferences.LanguageChannel.*;
 
 public class ChatPrefs {
 
+  @VisibleForTesting
+  public static final ImmutableMap<Locale, LanguageChannel> LOCALE_LANGUAGES_TO_CHANNELS = ImmutableMap.<Locale, LanguageChannel>builder()
+      .put(Locale.FRENCH, FRENCH)
+      .put(Locale.GERMAN, GERMAN)
+      .put(new Locale("ru"), RUSSIAN)
```

Comment:
There is no constant for "ru" that seems strange
and what is "be"?

Reply:
There are no constants for most countries/languages, only for some very specific ones. `be` is Belarussian.

Reply:
oh ok, so be it